lib/sass/calcite-web/components/_card.scss

```diff
@@ -43,6 +43,9 @@
 
 @mixin card-image-caption () {
   background: $opaque-white;
+  color: $off-black;
```

Comment:
unfortunately they are coming from `figcaption` styles in calcite. We could probably more those styles to a class (like buttons)